components/ButtonBar/examples/ButtonBarCustomizedFontExample.m

```diff
@@ -30,22 +29,30 @@ - (id)init {
   if (self) {
     self.colorScheme =
         [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
-    self.typographyScheme = [[MDCTypographyScheme alloc] init];
-    
+    self.typographyScheme =
+        [[MDCTypographyScheme alloc] initWithDefaults:MDCTypographySchemeDefaultsMaterial201804];
+
     self.title = @"Button Bar";
   }
   return self;
 }
 
+- (MDCContainerScheme *)containerScheme {
```

Comment:
_Nit:_ It seems odd to me to have the color and typography scheme be properties but the container scheme be computed. I suggest consolidating one way or the other.

Reply:
This is unfortunately due to how we're injecting schemes into our examples; namely that we don't know when the schemes will be set. I saw two approaches:

1. Implement didSet on both of the properties and update the container scheme each time.
2. Implement a getter that builds the scheme on demand.

2 felt like less overhead, while 1 would probably be a generally-better solution to support ongoing runtime theming changes. Supporting runtime theming changes isn't a particularly important priority right now or for this change though, so I went with the lower-overhead approach.